content/about/myths.md

```diff
@@ -21,6 +21,37 @@ In "typical" parsers, XML and JSON perform more or less identically in practice.
 
 But even ignoring the high performance, specialist end, XML still holds its own. Consider [these findings](https://www.balisage.net/Proceedings/vol10/html/Lee01/BalisageVol10-Lee01.html), for example.
 
+## XMPP is not secure.
+
+The hypothesis:
+> XMPP is not secure and does not implement modern technology.
```

Comment:
These are two separate hypotheses that are not necessarily related to one another.

I think this should be shortened to just `XMPP  is not secure` as this is what the subsequent text appears to be about.